src/components/sbb-datepicker/sbb-datepicker.tsx

```diff
@@ -203,10 +206,10 @@ export class SbbDatepicker implements ComponentInterface {
 
   private _formatValue(value: string): string {
     const match: RegExpMatchArray = value?.match(FORMAT_DATE);
-    if (!match || match.length <= 2) {
+    if (!match || match.length <= 2 || !match[2] || !match[3]) {
+      this._inputElement.dataset.sbbInvalid = 'true';
```

Comment:
For this logic I don't find tests, or do I miss something?

Reply:
I'll implement one